lib/octocatalog-diff/catalog-util/command.rb

```diff
@@ -68,7 +68,8 @@ def setup
         # enc?
         if @options[:enc]
           raise Errno::ENOENT, "Did not find ENC as expected at #{@options[:enc]}" unless File.file?(@options[:enc])
-          cmdline << "--node_terminus=exec --external_nodes=#{Shellwords.escape(@options[:enc])}"
+          cmdline << "--node_terminus=exec"
```

Comment:
Our rubocop rules will flag this as needing single quotes (mostly trying to keep the codebase consistent, and chose to use format similar to Puppet). If you could fix that up, this looks fine to me and I'll merge. Thanks!